protocols/gossipsub/src/error.rs

```diff
@@ -120,3 +120,37 @@ impl From<std::io::Error> for PublishError {
         PublishError::TransformFailed(error)
     }
 }
+
+/// Error associated with Config building.
+#[derive(Debug)]
+pub enum ConfigBuilderError {
```

Comment:
Is this re-exported in `lib.rs`?

Reply:
No its not.

Reply:
It should be because it is part of the public API!

Reply:
Okay